lib/utils.js

```diff
@@ -35,7 +36,7 @@ function mkdirpSync (dirpath) {
   }
 }
 
-function runCommand (command, args, ignoreStdError) {
+function runExecutable (command, args, ignoreStdError) {
   return new RSVP.Promise(function (resolve, reject) {
     var child = spawn(command, args, {
```

Comment:
@ro0gr Does this issue https://github.com/ciena-blueplanet/ember-cli-visual-acceptance/issues/92 still occur here when executing `git`? Even though I doubt anyone would run a CI build on Windows.